src/SFA.DAS.SelfService.Web/Views/Whitelist/Index.cshtml

```diff
@@ -2,28 +2,46 @@
     ViewData["Title"] = "Home";
     @model WhitelistViewModel
     @using SFA.DAS.SelfService.Web.Configuration
+
+    var ipAddressInvalid = !ViewData.ModelState.IsValid
+                            && ViewData.ModelState.ContainsKey("IpAddress") &&
+                            ViewData.ModelState["IpAddress"].Errors != null &&
+                            ViewData.ModelState["IpAddress"].Errors.Any();
 }
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">
             DAS Whitelist
         </h1>
+        <p class="govuk-body">To Whitelist your IP Address on DAS DEV &amp; PP Servers enter your details below.</p>
```

Comment:
Probably should work out a nicer description for this.

Could be sort and sweet but with a details component?
https://design-system.service.gov.uk/components/details/

'What does this give me access to?' for example